examples/hosts/iframe-host/src/containerProxy.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Comlink from "comlink";
+import { IContainer } from "@fluidframework/container-definitions";
+import { IRequest, IResponse } from "@fluidframework/core-interfaces";
+import { IQuorum } from "@fluidframework/protocol-definitions";
+
+export interface IContainerProxy {
+    attach(request: IRequest): Promise<void>;
+    request(request: IRequest): Promise<IResponse>;
```

Comment:
we shouldn't proxy request, as request generally returns object which can't cross the iframe boundary. someday we may want a proxy for that stuff, but for now we should just punt on it